pkgs/applications/video/vlc/default.nix

```diff
@@ -14,6 +14,7 @@
 , skins2Support ? !onlyLibVLC, freetype
 , removeReferencesTo
 , chromecastSupport ? true, protobuf, libmicrodns
+, srtSupport ? true, srt
```

Comment:
srt is pretty small. We can remove this flag.

Reply:
Also, SRT is becoming more and more popular and is a solid replacement for RTMP. Thanks, I would've also suggested this.